Sources/MediaViewController.swift

```diff
@@ -105,6 +109,19 @@ public class VLCMediaViewController: UICollectionViewController, UISearchResults
     func setupNavigationBar() {
         navigationItem.leftBarButtonItem = UIBarButtonItem(title: NSLocalizedString("SORT", comment: ""), style: .plain, target: self, action: #selector(sort))
     }
+    
+    func setupEmptyBackgroundView() {
+        emptyView = VLCEmptyLibraryView.init()
+        emptyView?.frame = view.bounds
+    }
+    
+    func displayEmptyViewIfNeeded() {
+        guard let view = emptyView, collectionView?.numberOfItems(inSection: 0) == 0 else {
```

Comment:
since the emptyview will be lazy it's not optional anymore and you can simply write 
collectionView?.backgroundView = collectionView?.numberOfItems(inSection: 0) == 0 ? emptyView : nil